src/plugins/paced-transcript/index.js

```diff
@@ -23,30 +25,35 @@ function pacedTranscript(config) {
 
   // Load the transcription file
   const initTranscript = async () => {
+    let transcriptResponse;
     try {
-      const transcriptResponse = await fetch(options.src);
-      const transcriptData = await transcriptResponse.json();
-      const captions = parseTranscript(transcriptData);
-
-      const captionsTrack = player.addRemoteTextTrack({
-        kind: options.kind,
-        label: options.label,
-        srclang: options.srclang,
-        default: options.default,
-        mode: options.default ? 'showing' : 'disabled'
-      });
-
-      // required for Safari to display the captions
-      // https://github.com/videojs/video.js/issues/8519
-      await new Promise(resolve => setTimeout(resolve, 100));
-
-      captions.forEach(caption => {
-        captionsTrack.track.addCue(new VTTCue(caption.startTime, caption.endTime, caption.text));
-      });
-
+      transcriptResponse = await fetch(options.src);
+      if (!transcriptResponse.ok) {
+        throw new Error(`loading transcription from ${options.src} failed, trying fallback URL`);
+      }
     } catch (error) {
-      console.error('Error loading transcription file:', error);
+      console.error(error);
+      transcriptResponse = await fetch(options.fallbackSrc);
```

Comment:
Please note that you are not catching the error here... i do not know what will happen when it gets propagated .. maybe it crashes the VP if so is that what we want?